osquery/tables/system/windows/shimcache.cpp

```diff
@@ -94,8 +94,11 @@ auto parseShimcacheData(const std::string& token,
 
   // Check to make sure the shimcache entry is greater than zero
   if (!string_path.empty()) {
-    // Get last modified time of the file
-    shimcache.last_modified = littleEndianToUnixTime(shimcache_time);
+    // Sometimes Shimcache artifacts have 0 as timestamp, if so skip filetime
+    // converstion
+    shimcache.last_modified = (shimcache_time == "0000000000000000")
```

Comment:
This is a nit.... Looking at this together I wonder if `if (!string_path.empty() || shimcache_time == "0000000000000000"){` is cleaner

Reply:
I took another look at this a decided to split the logic a bit.
Instead of checking if the string_path is not empty, i added an earlier check in the function: if the path size is zero then return early
https://github.com/osquery/osquery/blob/ee438db26c0e2519f9e3083d281a6efa95b50d06/osquery/tables/system/windows/shimcache.cpp#L59-L66

I left the ternary operator in